### DISCUSSION TITLE
Automatic RunPod installer script and full tutorial

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [lllyasviel/Fooocus#927] [Show and tell]
For those who needs I just released it with necessary instructions

[Fooocus SD Web UI RunPod Auto Installer - 1 Click - Latest Version](https://www.patreon.com/posts/fooocus-sd-web-1-92759045)

Automatic installer for Stable Diffusion Fooocus Web UI on RunPod. 1 Click. Use Fooocus on RunPod with all models and features. Latest version. Follow the instructions on the Patreon post.

And here my public tutorial for Fooocus

[**Fooocus Stable Diffusion Web UI - Use SDXL Like You Are Using Midjourney - Easy To Use High Quality**](https://youtu.be/jHTkVm2mcfs)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/zuJst_wZ7_qs_2e3KV33W.png)](https://youtu.be/jHTkVm2mcfs)

$0.34/hr price having RTX 3090 pod gives 3.7 it/s with 1024x1024 resolution 

![image](https://github.com/lllyasviel/Fooocus/assets/19240467/2ffd3b4b-2dd2-4874-b0a4-bcbcd468718d)

**Comment by kubilaykilinc** [comment]
Furkan burada bu tip promosyonel şeyleri sevmeyebilirler. Sen bilirsin
